test/jb_projects/challenge_handle.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { getTimestampFn } from '../helpers/utils'
+
+describe('JBProjects::challengeHandle(...)', function () {
+
+  let jbOperatorStoreFactory;
+  let jbOperatorStore;
+
+  let jbProjectsFactory;
+  let jbProjectsStore;
+
+  let deployer;
+  let projectOwner;
+  let addrs;
+
+  let projectHandle = "PROJECT_1";
+  let projectHandleNotTaken = "PROJECT_2"
+  let projectHandleNotTaken2 = "PROJECT_3"
+  let metadataCid = "";
```

Comment:
These should be const and the variable names should be capitalized snake case.

PROJECT_HANDLE, PROJECT_HANDLE_NOT_TAKEN1, PROJECT_HANDLE_NOT_TAKEN2